metadata_backend/api/handlers.py

```diff
@@ -158,6 +158,65 @@ async def delete_object(self, req: Request) -> Response:
                  f"in schema {schema_type} was successful.")
         return web.Response(status=204)
 
+    async def get_folders(self, req: Request) -> Response:
+        """Get all possible object folders from database.
+
+        :param req: GET Request
+        :returns: JSON list of folders
+        """
+        raise web.HTTPNotImplemented
+
+    async def post_folder(self, req: Request) -> Response:
+        """Save object folder to database.
+
+        :param req: POST request
+        :returns: JSON response containing folder ID for submitted object
+        """
+        db_client = req.app['db_client']
+        operator = Operator(db_client)
```

Comment:
I don't think folder operations need to be proxyed through operator-class. Or at least then you should create some kind of FolderOperator, since the current operator class logic is done for xml/json metadata objects.

It's also more logical if folders are in database called "folders", so my suggestion would be to add here: `db_service = DBservice("folders", db_client)` and do db operations with it.

Reply:
second this, i think it should be in different db collection